torch/nn/qat/__init__.py

```diff
@@ -4,4 +4,15 @@
 This package is in the process of being deprecated.
 Please, use `torch.ao.nn.qat.dynamic` instead.
 """
+from . import dynamic  # noqa: F403
```

Comment:
Do we need to do the same thing in `torch/nn/qat/dynamic/__init__.py`? Does this work right now: `import torch; torch.nn.qat.dynamic.Linear`?

Reply:
Same for `torch/nn/quantized/dynamic/__init__.py`

Reply:
We don't need to do it for the `dynamic/__init__.py`, but that raises a point on using the `torch.nn.qat.dynamic.Linear` -- I will double check

Reply:
You were right, the `qat` was not visible to the users. Fixed